tests/page-templates/17-usertiming/01-usertiming-basic.html

```diff
@@ -0,0 +1,24 @@
+<%= header %>
+<%= boomerangSnippet %>
+<script src="01-usertiming-basic.js" type="text/javascript"></script>
+<script src="../../vendor/usertiming-compression/src/usertiming-decompression.js" type="text/javascript"></script>
```

Comment:
That's where the bower components get installed.
Changing it would require changing all the tests.